File: lib/src/main/java/com/raqun/sensitiveactivity/ScreenOrientations.java
package com.raqun.sensitiveactivity;

/**
 * Created by tyln on 19.02.2018.
 */

public final class ScreenOrientations {

    private ScreenOrientations() {
        // No instance available
    }

    public static final float[][] orientations = {
            {9.81f, 0, 0},                                                             // LandscapeLeft
            {-9.81f, 0, 0},                                                            // LandscapeRight
            {0, -9.81f, 0},                                                            // Portrait
            {0, 9.81f, 0},                                                             // UpsideDown
            {0, 0, -9.81f},                                                            // FaceUp
            {0, 0, 9.81f}                                                              // FaceDown
    };
}

File: live-orientation/src/main/java/com/raqun/live_orientation/ScreenOrientations.kt
package com.raqun.sensitiveactivity

/**
 * Created by tyln on 19.02.2018.
 */

object ScreenOrientations {

    val orientations = arrayOf(floatArrayOf(9.81f, 0f, 0f), // LandscapeLeft
            floatArrayOf(-9.81f, 0f, 0f), // LandscapeRight
            floatArrayOf(0f, -9.81f, 0f), // Portrait
            floatArrayOf(0f, 9.81f, 0f), // UpsideDown
            floatArrayOf(0f, 0f, -9.81f), // FaceUp
            floatArrayOf(0f, 0f, 9.81f)                                                              // FaceDown
    )
}
